posthog/models/exported_asset.py

```diff
@@ -44,8 +44,9 @@ class ExportFormat(models.TextChoices):
         PNG = "image/png", "image/png"
         PDF = "application/pdf", "application/pdf"
         CSV = "text/csv", "text/csv"
+        EXCEL = "text/xlsx", "text/xlsx"
```

Comment:
This is a shortcut, because the longer real mimetype doesn't fit unless we adjust the field max_length. Do we aim to use real mimetypes according to spec here or can we be flexible?

Reply:
I think that's fine here since the type is used only internally and not as HTTP-header (?).

Reply:
It's 100% safe to extend the max length (it's an instant operation in Postgres) so I would lean towards just doing that and having the _real_ type 😄

Reply:
`application/vnd.openxmlformats-officedocument.spreadsheetml.sheet`

Cannot use that in the UI though 🙈

Reply:
Oof, that's an impressive one